coordinateutils/src/HealpixSkyMap.cxx

```diff
@@ -894,6 +920,24 @@ G3SkyMapPtr HealpixSkyMap::Rebin(size_t scale, bool norm) const
 	return out;
 }
 
+void HealpixSkyMap::SetShiftRa(bool shift)
+{
+	assert(!IsRingSparse());
+
```

Comment:
The idea here is that it only works if you aren't already in this representation? The semantics are a little weird that the circumstances under which this can be set (and read) I think are only ones in which this global parameter does nothing. Not quite sure what to do about that, though...

Reply:
I guess I could make this function explicitly rearrange the ringsparse representation if you change the parameter?  It's a bit of bookkeeping, but doable.  Then it doesn't matter when you set it, and makes switching back and forth to test things out a bit more straightforward.

Reply:
Done